From a7e07900f90e3ae68e8ef25e4fbd4589b86dff68 Mon Sep 17 00:00:00 2001
From: Ma Ling <ling.ma@intel.com>
Date: Fri, 6 Feb 2009 09:14:15 +0800
Subject: [PATCH 04/20] Don't disable vga centering bit.

commit id b9f5915ce812144ffd9d2aa42e8ba856129c35e,
which resolved bug #17235, but generate new regression-bug #19715.
This patch intends to resolve bug #17235, and avoid regression as well.
We have successfully re-tested it for bug #17235 and #19715 respectively.
---
 src/i830_display.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git xf86-video-intel-2.4.3/src/i830_display.c xf86-video-intel-2.4.3/src/i830_display.c
index 49eb1b5..bf2c530 100644
--- xf86-video-intel-2.4.3/src/i830_display.c
+++ xf86-video-intel-2.4.3/src/i830_display.c
@@ -798,10 +798,6 @@ i830_disable_vga_plane (xf86CrtcPtr crtc)
 
     vgacntrl |= VGA_DISP_DISABLE;
 
-    /* disable center mode */
-    if (IS_I9XX(pI830))
-	vgacntrl &= ~(3 << 24);
-
     OUTREG(VGACNTRL, vgacntrl);
     i830WaitForVblank(pScrn);
 
-- 
1.6.2

